examples/js/loaders/AMFLoader.js

```diff
@@ -48,18 +48,18 @@ THREE.AMFLoader.prototype = {
 
 	parse: function ( data ) {
 
-		function loadDocument( data ) {
+		function _loadDocument( data ) {
```

Comment:
> If I wanted to allow users to specify a default material...

I do not think we do that with loaders. The user can change the material in the loader callback.